connectors/src/connectors/slack/bot.ts

```diff
@@ -246,11 +246,20 @@ async function botAnswerMessage(
     throw new Error("DUST_FRONT_API environment variable is not defined");
   }
 
+  let requestedGroups: string[] | undefined = undefined;
+
+  if (slackUserInfo.is_bot) {
+    const botName = slackUserInfo.real_name || slackUserInfo.display_name;
```

Comment:
```suggestion
    const botName = slackUserInfo.real_name ?? slackUserInfo.display_name;
```

Reply:
Honestly I never understand why ?? is better than || in this case. Do we have a convention in the code base? I mean here it really means one or the other.